src/app/page.js

```diff
@@ -82,7 +95,7 @@ export default function Page() {
           </aside>
         </>
       </SideNav>
-      {alert && <Toast type={alert} message={message} />}
+      {showAlert && <Toast type={alert} message={message} />}
```

Comment:
Then this becomes `alert`

```suggestion
      {alert && <Toast type={alert} message={message} />}
```